.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+{
+"build": { "dockerfile": "Dockerfile" },
+"forwardPorts": [
+	8080
+],
+"extensions": [
+	"nadako.vshaxe",
+	"ms-azuretools.vscode-docker",
+	"openfl.lime-vscode-extension"
+]
+}
```

Comment:
JSON formatting is inconsistent with every other tab-indented JSON file in this repository, AFAIK.